libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -425,6 +425,15 @@ - (NSURL *)frontDoorUrlWithReturnUrl:(NSString *)returnUrlString returnUrlIsEnco
     NSURL *instUrl = creds.apiUrl;
     NSString *fullReturnUrlString = returnUrlString;
     
+    // Catch if we have a absoute URL, if we do then url decode to avoid
+    // errors with 'Invalid page redirection'
+    if(createAbsUrl && [returnUrlString hasPrefix:@"http"]) {
```

Comment:
Shouldn't this be `https`? I don't think any pages within the `force.com` domain support `http`.

Reply:
It was just the same as the line below - will catch both https and http